src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/SymbolFactory.vb

```diff
@@ -49,6 +49,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             Return False
         End Function
 
+        Friend Overrides Function IsConstModifierType(moduleSymbol As PEModuleSymbol, type As TypeSymbol) As Boolean
+            ' VB doesn't deal with ref-readonly parameters or return-types.
+            Return False
```

Comment:
Will rename both to `IsUnderstoodXXXXModifierType`

---
In reply to: [118021028](https://github.com/dotnet/roslyn/pull/19658#discussion_r118021028) [](ancestors = 118021028)